src/Worker/Cron.php

```diff
@@ -132,6 +134,10 @@ public static function execute()
 				Worker::add(Worker::PRIORITY_LOW, 'OptimizeTables');
 			}
 
+			foreach (User::getList($start=1, $limit=PHP_INT_MAX, $type='active') as $user) {
```

Comment:
These variables are initialized but never used, what was the point?

Reply:
`User::getList` usually gets users beginning with a `uid` of 0, which would be the system user that does not need a homepage check. And normally it would return chunks of the user base controlled by the `limit` parameter. All `active` users should be checked once a day by the cron job, so I've set the chunk size so a really large number.

From the returned array, only the users `uid` is interesting though. Which is used for the new Worker class.

Reply:
Thanks for the answer, however I'm more interested in the `$start=` part, this is initializing a variable named `$start` before passing its value as first parameter of `User::getList`, but then the variable itself isn't used. Here's what a regular call would look like:
```suggestion
			foreach (User::getList(1, PHP_INT_MAX, 'active') as $user) {
```

Reply:
I guess then the answer is, habit from python and that I was not aware that this would initialize the variable on the local scope.

Reply:
It's an interesting way of showing the names of the parameters when calling a method, I was wondering if it was the goal.

Reply:
Yes, it's explicitly naming the parameters. You don't have to, but it does not hurt either and sometimes it is clearer to write them down.